theme.json

```diff
@@ -391,6 +391,15 @@
 					}
 				}
 			},
+			"core/image": {
+				"variations": {
+					"rounded": {
+						"border": {
+							"radius": "16px"
```

Comment:
I think this change is interesting to consider for a future refactor of the patterns @richtabor

Reply:
Yea I'm a bit torn. I kinda think the default radius should be what all the images in the theme are; so you don't have to apply it to others. But I think it could go either way.

Reply:
It's a few steps to revert the change when you don't need it. We have a pattern with logos that looks bad if we do this and forget to undo the radius

Reply:
> It's a few steps to revert the change when you don't need it. We have a pattern with logos that looks bad if we do this and forget to undo the radius

It could be done at the pattern level though; to remove border-radius. Instead of having to apply border radius for every image added. @colorful-tones do you mind moving the variations.rounded.border.radius change to a separate PR for testing. I'd love to try it out and see if that works (to then add rounded variation to most images within the patterns).

Reply:
I'm just working off of Figma designs is all. 
![Screenshot 2023-09-05 at 4 50 50 PM](https://github.com/WordPress/twentytwentyfour/assets/405912/ad4b0d5e-50bb-47e6-b85a-8d9c168e00db)

Reply:
Right, but we apply a deeper level of integration/consistency with the block editor :)

Reply:
> @colorful-tones do you mind moving the variations.rounded.border.radius change to a separate PR for testing.

> I'd love to try it out and see if that works (to then add rounded variation to most images within the patterns).

Oh it works @richtabor . Did we just think I dropped it in for fun? 🙃